controls/sshd_spec.rb

```diff
@@ -207,9 +207,9 @@
 control 'sshd-16' do
   impact 1.0
   title 'Server: Use privilege separation'
-  desc 'UsePrivilegeSeparation is an option, when enabled will allow the OpenSSH server to run a small (necessary) amount of code as root and the of the code in a chroot jail environment. This enables ssh to deal incoming network traffic in an unprivileged child process to avoid privilege escalation by an attacker.'
+  desc 'UsePrivilegeSeparation is deprecated.'
   describe sshd_config(sshd_custom_path + '/sshd_config') do
-    its('UsePrivilegeSeparation') { should eq(sshd_valid_privseparation) }
```

Comment:
I think we also need to remove https://github.com/dev-sec/ssh-baseline/blob/0932d5fe8789d33265eb277d543a9c08e8c79089/controls/sshd_spec.rb#L35-L39 and https://github.com/dev-sec/ssh-baseline/blob/3849c52ed8f6fe17d82c097c2428d0af04282bb5/libraries/ssh_crypto.rb#L172-L210